SEED-platform/seed
# Demo data with audit logs

#### Any background context you want to provide?
This should fix the issues with audit log data in the command for creating sample data.  Had a problem merging develop into the branch I had been working on so since it is only two files I just made a new branch.  This PR does not address issues with using the generated data as import files.
#### What's this PR do?
Adds a management command called create_sample_data that takes the following command-line params:
--A number of A cases to create
--B number of B cases to create
--C number of C cases to create
--D number of D cases to create
--Y years to be created in a csv string.  E.G. "2014,2015,2016"
--audit-depth the number of additional records to be created for each case for each year.  E.G. "--A 1 --B 0 --C 0 --D 0 --Y "2014,2015,2016" --audit-depth 5" will create 6 PropertyStates and 6 TaxlotStates (remember case A is one property to one taxlot) for each of 2014, 2015, and 2016.  Each year creates an additional 5 states to be used in the audit logs for a total of 6.  But there is only one Property and Taxlot view record created for each year.
#### How should this be manually tested?
Visually using the new UI changes.  Make sure all of the cases seem to agree in the building list and the audit log information looks reasonable in the detail view.
#### What are the relevant tickets?
#### Screenshots (if appropriate)
#### Definition of Done:
NA I think.  This is generating sample data for use in development, it is not actually used in production.
